src/NzbDrone.Core/Extras/Subtitles/SubtitleFile.cs

```diff
@@ -1,10 +1,28 @@
-﻿using NzbDrone.Core.Extras.Files;
+﻿using System.Collections.Generic;
+using System.IO;
+using NzbDrone.Core.Extras.Files;
 using NzbDrone.Core.Languages;
 
 namespace NzbDrone.Core.Extras.Subtitles
 {
     public class SubtitleFile : ExtraFile
     {
         public Language Language { get; set; }
+
+        public string AggregateString => Language + Extension + LanguageTags;
```

Comment:
Make it `Language + LanguageTags + Extension` to match naming?

Reply:
Reordered in 49d89e2.